client/.env

```diff
@@ -1 +1,2 @@
 EXTEND_ESLINT=true
+REACT_APP_GOOGLE_KEY=AIzaSyAeoBJPdX8fHzuIxXA6vIknwyceEuut3tY
```

Comment:
I don't know if there is a better way to use the google key, @Zaydme  any ideas?

Reply:
@ddveloper @Zaydme I've left a DM for Zaydme in regards to this.   Right now, I'm writing an api to encrpt the key from the server, and remove this env variable from the FE so that the client can decrypt it when it uses the key.   (using CryptoJS).    Since this exposes the key to devtools even if the env file is hidden.   Also I have Dawei Zhang researching for verification on this solution.   Will commit these changes when I get this portion done on the backend.

Reply:
add @mannykary

Reply:
It looks like we need a "browser" key for this, (keeping the backend key the same).   So I think you were right about two different keys.   
https://webmasters.stackexchange.com/questions/95918/restricting-google-maps-js-api-keys-to-just-my-domain

I think we need to ask Josh Morel or (Manny possibly) to create a new key and add the domain restriction(s).  Until they respond, I think we're ok leaving the FE env variable (key) in place for just the PR to test.

Reply:
@maskowe you can and we should restrict based on referrer domain which is set in Google Maps Cloud console. The key shouldn't be in source control anyways but you dont need to encrypt it.. (due to referrer limitation). Check out what is done with GTM key (cc @mannykary )

Reply:
@maskowe See https://github.com/FightPandemics/FightPandemics/blob/staging/.github/workflows/production-workflow.yml#L26-L29. For the GTM key, we added it to Github Secrets and then loaded it in when the client Docker image is built. We will need a Google Key for each environment (Review, Staging, and Production). You can DM me or anyone with admin repo access (I believe @joshmorel, @ddveloper, and @Zaydme have access) so that the keys can be added to Github Secrets.

Reply:
@mannykary Thanks - Josh offered to help soon.   If he's unable for time restrictions, I'll let you know.

Reply:
@mannykary Josh M. has added the three browser keys we need as:

created the 3 keys in google cloud console:
FE Maps API Key - Staging: AIzaSyApSv0_W3aY9H0YISFXTIyqcIFWDUBQ6PI
FE Maps API Key - Review: AIzaSyCmPYn3QSsnS7Xb4Hi4Vqe_vCAyndfWvzI
FE Maps API Key - Production: AIzaSyC9qjNSbcpULY05SezUBooRXvJ0iChU-g8 (edited

Can you possibly get the time to add these to github secrets?

He has added the keys as values to corresponding GitHub secrets keys:
STAGING_FE_GOOGLE_MAPS_API_KEY
REVIEW_FE_GOOGLE_MAPS_API_KEY
PRODUCTION_FE_GOOGLE_MAPS_API_KEY

And I will do the yml changes needed.   
Thanks again.

Reply:
Or you can DM me the link and credentials with instructions so that I can try to do this myself, if time is an issue.

Reply:
Keys have been addressed for all branches.   Please retest at https://feature1201-display-health-facilities.fightpandemics.xyz